src/bitmessagekivy/baseclass/addressbook.py

```diff
@@ -0,0 +1,171 @@
+# pylint: disable=unused-argument, consider-using-f-string, import-error
+# pylint: disable=unnecessary-comprehension, no-member, no-name-in-module
+
+"""
+addressbook.py
+==============
+
+All saved addresses are managed in Addressbook
+
+"""
+
+from functools import partial
+
+from kivy.clock import Clock
+from kivy.properties import (
+    ListProperty,
+    StringProperty
+)
+from kivy.uix.screenmanager import Screen
+
+from helper_sql import sqlExecute
+
+import state
+
+from bitmessagekivy.get_platform import platform
+from bitmessagekivy import kivy_helper_search
+from bitmessagekivy.baseclass.common import (
+    avatarImageFirstLetter, toast,
+    ThemeClsColor, SwipeToDeleteItem
+)
+from bitmessagekivy.baseclass.popup import AddbookDetailPopup
+from bitmessagekivy.baseclass.addressbook_widgets import HelperAddressBook
+
+
+class AddressBook(Screen, HelperAddressBook):
+    """AddressBook Screen class for kivy Ui"""
+
+    queryreturn = ListProperty()
+    has_refreshed = True
+    address_label = StringProperty()
+    address = StringProperty()
+
+    def __init__(self, *args, **kwargs):
+        """Getting AddressBook Details"""
+        super(AddressBook, self).__init__(*args, **kwargs)
+        self.addbook_popup = None
+        Clock.schedule_once(self.init_ui, 0)
+
+    def init_ui(self, dt=0):
+        """Clock Schdule for method AddressBook"""
+        self.loadAddresslist(None, 'All', '')
+        print(dt)
```

Comment:
if you really need this printed out, use `logger.debug` or something like that, not `print`